diff --git a/firmware/l0dable/dbgmesh.c b/firmware/l0dable/dbgmesh.c
index fdb4200..6761c4c 100644
--- a/firmware/l0dable/dbgmesh.c
+++ b/firmware/l0dable/dbgmesh.c
@@ -143,7 +143,7 @@ void m_time(void){
         lcdPrint("N:");
         lcdPrint(IntToStr(meshnice,3,0));
         lcdPrint(" P:");
-        lcdPrintInt(GLOBAL(privacy));
+        lcdPrint(IntToStr(GLOBAL(privacy),1,0));
 
 	lcdPrint(" R:");
 	lcdPrint(IntToStr(getrelease()%0xfff,3,F_HEX));
